src/reference/config/writing-platform-configs.rst

```diff
@@ -49,7 +49,7 @@ Why Were Platforms Introduced?
 
 - Allow a compute cluster with multiple login nodes to be treated as a single
   unit.
-- Allow Cylc to elegantly handle failure of to communicate with login nodes.
+- Allow Cylc to elegantly handle failure to communicate with login nodes.
 - Configure multiple platforms with the same hosts; for example you can use
   separate platforms to submit jobs to a batch system and to background on
   ``localhost``.
```

Comment:
I don't think this a valid reason why platforms were introduced - this was just as easy before.
Instead we could add:
```
- Reduce the number of ssh connections required for job submission and polling.
```